### PR TITLE
Fix/import

### PR DESCRIPTION
**Pull request by alexeyMohnatkin**
This change is [Reviewable](https://reviewable.io/reviews/shakacode/sass-resources-loader/45)

### CODE REVIEW

**Comment by alexeyMohnatkin** [comment]
#40

**Comment by justin808** [comment]
@alexfedoseev Please review.

File: src/utils/rewriteImports.js [hunk 1]
```diff
@@ -14,11 +14,24 @@ export default (error, file, contents, moduleContext, callback) => {
     return callback(null, contents);
   }
 
+  const getNewImportPath = (oldImportPath) => {
+    const absoluteImportPath = path.join(path.dirname(file), oldImportPath);
+
+    // if import path starts with / resolve from project root or webpack.context
+    if ((/^\//).test(oldImportPath)) {
+      return path.join(rootContext, oldImportPath);
+    }
```
Comment (alexfedoseev) [inline src/utils/rewriteImports.js]: Is this a common pattern to write imports like this? It looks confusing.

File: src/loader.js [hunk 2]
```diff
@@ -48,6 +48,8 @@ module.exports = function(source) {
     );
   }
 
+  const rootContext = webpack.options.context || process.cwd();
```
Comment (alexfedoseev) [inline src/loader.js]: There is `webpackConfigContext` on line 42.

**Comment by kieran** [comment]
can we release this fix to npm?

**Comment by alexfedoseev** [comment]
@kieran Published `1.3.1`.

**Pull request merged**